### ISSUE TITLE
Usama's Feedback - 04-March-2019

### ISSUE THREAD

**Issue by ZarshaB** [asifjans/Inovedia#367]

**Comment by ZarshaB** [comment]
1>>	Please refer to attached images. The font used in template and the reference provide earlier is different. Please use the tool “Whatfont” to see the font type, size and other specification used in references provided earlier. It should be exactly the same for each line on each page through out the site.

**Comment by ZarshaB** [comment]
2>> When I sign in, the dividing orange line, my name and the cart should be moved slightly towards right.

**Comment by ZarshaB** [comment]
3>> The icons of How it works section on homepage (both buyer and seller) needs to be reduced in size. The look very big as compared to the actual shared images.

**Comment by ZarshaB** [comment]
4>> In drop down menu of against my name where we have My Profile, My Favorites etc, the gap between these lines should be increased and font a little smaller to make them look sharp.

**Comment by ZarshaB** [comment]
5>> Please ensure that all the links, meta tags and other description while you take cursor to the tab on google chrome should be as per the older version. We don’t’t want to effect our SEO.

**Comment by ZarshaB** [comment]
6>> I don’t see featured cars although I made some cars featured from admin.

**Comment by ZarshaB** [comment]
7.	Please change the wordings for HIW on home page as follows:-
How it works for Seller 
 
Register your account 
Create your account on WelaDela

 
Post an ad 
Post a FREE ad to include your car in raffle draw
 
Ad goes LIVE 
Including a FREE care valuation from WelaDela
 
Sell Fast and get more 
We market your car everywhere FREE to ensure sale in quick sale

**Comment by ZarshaB** [comment]
8>>  Please change the wordings for HIW on home page as follows:-
FOR BUYERS
Register your account 
Create your account on WelaDela
 
Browse car listings 
Find used car for sale in UAE and check their available items
 
Get your big ticket 
Shop for your chosen used cars and receive your raffle draw ticket
 
Get Lucky 
Win the used cars in lucky draw and claim your new ride

**Comment by ZarshaB** [comment]
9>> Download our App
Thinking of getting a new car?
Download our free app to win your dream car

**Comment by ZarshaB** [comment]
10>> The social media icons should be colored. And Youtube icon should be aligned in center with others.

**Comment by ZarshaB** [comment]
11>>Under We accept heading and Master card, Visa and COD icons, please add following two icons. They should be in line with icons above them

**Comment by ZarshaB** [comment]
12>> Login
Please refer to image # 3. Remove this heading and all similar headings (in grey bar) on all pages like View Cars, My Account etc.
![image](https://user-images.githubusercontent.com/35685146/53720419-8afcba80-3e82-11e9-881f-f2728e36f918.png)

**Comment by ZarshaB** [comment]
13>>Please remove the boxes of New Customer and Returning Customer. Instead use a green line under New Customer (and green color for this New Customer heading) and orange line under Returning customer headings (and orange color font for Returning customer heading). Reference of what I’m looking at can be taken from Car Description page where we have “Description” and “details” tabs and have smart lines under them. Attaching Image #4 for reference.

**Comment by ZarshaB** [comment]
14>> Same as above when Forgot password link is clicked

**Comment by ZarshaB** [comment]
15>> Login
When email address is submitted for Forgot password, the error message We're here to help. Password reset link has been sent to your email address butt_usama@hotmail.com should be a pop-up box(same as we are using through the site)

**Comment by ZarshaB** [comment]
16>> Login
Font type and size should be uniform across the site.

**Comment by ZarshaB** [comment]
17>> Log Out Message
The Log Out Message  OK. You’ve been logged out. 
Until next time, stay awesome and come back soon!!! should be a pop-up message

**Comment by ZarshaB** [comment]
18>> My Favorites
“View Cars” button on empty My Favorites is opening up old View cars page

**Comment by ZarshaB** [comment]
19>> 
My Ads

“Submit an Ad” button on empty My Ads is opening up Submit ad format

**Comment by ZarshaB** [comment]
20>> Register

1.	Please remove the heading “Register Account”
2.	When I click on T&C against check box at the bottom, it opens old T&Cs page
3.	Register Button is not working

**Comment by ZarshaB** [comment]
21>> 1.	All the hyperlinks like in FAQs, T&Cs, Policies etc are opening up old pages

**Comment by ZarshaB** [comment]
22>> Please remove the Submit Ad button in grey bar on every other page. It should be only on View Cars Page & car detail page

**Comment by ZarshaB** [comment]
23>>	FONT SIZE AND TYPE

**Comment by ZarshaB** [comment]
24>> The general look and feel of the site is not as per the designed images. Probably because of the font type used on site is different and size is also a bit bigger.

**Comment by ZarshaB** [comment]
25>> All the messages that customer gets should be pop-up. The pop-up message lay out should be uniform and text type and size should be in line with rest of the site. Please show some design for a pop up message. You can take some from google search. Refer to image 5 attached. Too much spacing given on top and bottom. The new design should simple, minimal, smart and sleek.

**Comment by ZarshaB** [comment]
26>> Due to time constraints, I wont be able to fully check the functionalities and flows. Please make sure that everything is correctly working, all buttons are functional and all the messages and linked emails are properly sent and displayed.

**Comment by ZarshaB** [comment]
27>> How It Works

The text doesn’t look good. Please refer to homepage for placement of the text

**Comment by ZarshaB** [comment]
28>> Contact Us

•	The text under We do read our emails heading is going out of proportion. Please refer to

**Comment by ZarshaB** [comment]
29 >> Contact Us
•	The 2nd icon with land line number should be moved a little lower so it is proportionate with the text and all other icons.

**Comment by ZarshaB** [comment]
30>> Contact Us
•	In whatsapp number, format should be +971 50 456 3832

**Comment by ZarshaB** [comment]
31>> Contact Us
•	Font, placement and size should be as per the templates provided

**Comment by ZarshaB** [comment]
32>> View Cars - Grid
The price filter is not going below 100.

**Comment by ZarshaB** [comment]
33>> View Cars - Grid
In all ads displayed on main View Cars Page, the ad description should be long enough to cover full length of the image above it.

**Comment by ZarshaB** [comment]
34>> View Cars- Grid
The font for ad description should be black

**Comment by ZarshaB** [comment]
35>> The buttons showing number of pages at the bottom should be as per the template.

**Comment by ZarshaB** [comment]
36>>	Next Page button not working at the bottom.

**Comment by ZarshaB** [comment]
37>> The Get Ticket Button should be white background and orange text (sorry for changing it again but looks very cluttered)

**Comment by ZarshaB** [comment]
38>> Ticket Sold under each ad should be only Sold: 12.5%. Remove the word Tickets

**Comment by ZarshaB** [comment]
39>> 	All fonts should be reduced and type should be as per template

**Comment by ZarshaB** [comment]
40>> The distance between search filter column on left and car list should be increased.

**Comment by ZarshaB** [comment]
41>> When the cart has some items in it then the cart button on top doesn’t look neat with digits 1 or 2 in them. Very blurry. Please make it look clean with perfect circle.

**Comment by ZarshaB** [comment]
42>> The length and width of each ad should be reduced. Way too big. Please look at the template. The distance of white portion of the page on the right side is more.

**Comment by ZarshaB** [comment]
43>> The font of filters also need to be changed

**Comment by ZarshaB** [comment]
44>> 	There is big difference in look and feel of pages on site and the images shared with you earlier. For all pages. Open one site page in one tab and image provided by the designer of the same page in other tab and then you’ll see the difference I’m talking about.

**Comment by ZarshaB** [comment]
45>> View Cars - List

The price filter is not going below 100

**Comment by ZarshaB** [comment]
46>> In all ads displayed on main View Cars Page, ad description should be fully shown. However, if its way too long then it should be stopped till it reaches parallel to start of Get Ticket button.

**Comment by ZarshaB** [comment]
47>>	There are small boxes with very thin lines in template provided by designer whereas they are missing on website.

**Comment by ZarshaB** [comment]
48>>	The placement of ticket price is higher as compared to template provided by designer

**Comment by ZarshaB** [comment]
49>> Car Detail

The Get Ticket Button in car detail page should be orange background and white text (Leave it as it is. Only needs to be changed in car list and home page)

**Comment by ZarshaB** [comment]
50>>	Ticket Sold under each ad should be only Sold: 12.5%. Remove the word Tickets

**Comment by ZarshaB** [comment]
51>> All fonts should be reduced and type should be as per template

**Comment by ZarshaB** [comment]
52>>	The distance between search filter column on left and car list should be increased.

**Comment by ZarshaB** [comment]
53>> There is big difference in look and feel of pages on site and the images shared with you earlier. For all pages. Open one site page in one tab and image provided by the designer of the same page in other tab and then you’ll see the difference I’m talking about.

**Comment by ZarshaB** [comment]
54>> The quantity button is working in odd numbers and goes to negative as well.

**Comment by ZarshaB** [comment]
55>> The columns of Details should be reduced in width and the font size and type should be in accordance with the templates provided to you earlier.

**Comment by ZarshaB** [comment]
56>>	Don’t have add to favorites button in details page.

**Comment by ZarshaB** [comment]
57>> Check Out Page

Please remove the heading on top left in grey bar (View Cars/Check-out)

**Comment by ZarshaB** [comment]
58>> Quantity button not working

**Comment by ZarshaB** [comment]
59>> COD is also taking me to payment gateway

**Comment by ZarshaB** [comment]
60>>	When I click on check out without entering details like name etc, it is not prompting me for error.

**Comment by ZarshaB** [comment]
61>>	Pop-up messages lay out needs to be changed. Please refer to General section for details

**Comment by ZarshaB** [comment]
62>> Submit your Ad

	No option of removing the main image and other images as well.

**Comment by ZarshaB** [comment]
63>> No option of deleting the ownership documents.

**Comment by ZarshaB** [comment]
64>> Please use some paid image optimizer to optimize the image. Find some optimizer, which can reduce the file size as well as should make the resolution as per our format so that the images doesn’t become blurry.

**Comment by ZarshaB** [comment]
65>>	The check box on last page is smaller in size and is not in line with the text. Format is also different. Use the check box which we used on order/check-out page.

**Comment by ZarshaB** [comment]
**Reopened**
7 >> Please make "D" capital for WelaDela.
![image](https://user-images.githubusercontent.com/35685146/53788826-af20d000-3f44-11e9-9189-94e1ee814b86.png)

**Comment by ZarshaB** [comment]
**Reopened** 
11>> Please add verified visa with master card. They should be in one line. 
![image](https://user-images.githubusercontent.com/35685146/53789078-6ae1ff80-3f45-11e9-9a92-bbb52fd03b19.png)

**Comment by ZarshaB** [comment]
17>> Can you please also redirect My Rewards and Purchase History pages View Cars button to new site too?
![image](https://user-images.githubusercontent.com/35685146/53789379-125f3200-3f46-11e9-8f92-815db2b735f8.png)
![image](https://user-images.githubusercontent.com/35685146/53789397-19864000-3f46-11e9-902c-86b729465f91.png)

**Comment by ZarshaB** [comment]
**Reopened** 
20>> Register
Register Button does not work when I fill the form and try to submit it.

**Comment by ZarshaB** [comment]
**Reopened**
21>> It should open mailbox on clicking on it. 
![image](https://user-images.githubusercontent.com/35685146/53789729-f4460180-3f46-11e9-83be-d5a8163b8b04.png)

These are all redirecting to live site.
![image](https://user-images.githubusercontent.com/35685146/53789775-19d30b00-3f47-11e9-9d20-8e7b1930ce9b.png)
![image](https://user-images.githubusercontent.com/35685146/53789811-2f483500-3f47-11e9-87ea-e5b0c0a70b99.png)

**Comment by ZarshaB** [comment]
**Reopened**
29>> Space is not equal from above and lower of icon.
![image](https://user-images.githubusercontent.com/35685146/53790144-13915e80-3f48-11e9-98bd-2004dc12ee64.png)

**Comment by ZarshaB** [comment]
30>> 	In whatsapp number, format should be +971 50 456 3832
As he has asked for whatsapp number format, not for landline.
![image](https://user-images.githubusercontent.com/35685146/53790240-58b59080-3f48-11e9-9f6a-7fc2410d5ab6.png)

**Reference:**
![image](https://user-images.githubusercontent.com/35685146/53790279-6a973380-3f48-11e9-8080-0330e2a99806.png)

**Comment by ZarshaB** [comment]
32>> Now filter is going below 100. Can you please remove this blue outline around price filter button?
![image](https://user-images.githubusercontent.com/35685146/53790390-a631fd80-3f48-11e9-8572-241c7348110c.png)

**Comment by ZarshaB** [comment]
**Reopened**
Still there
33>> View Cars - Grid
In all ads displayed on main View Cars Page, the ad description should be long enough to cover full length of the image above it.

It should cover image length. 
![image](https://user-images.githubusercontent.com/35685146/53790598-235d7280-3f49-11e9-8c45-547004f12a89.png)

**Comment by Anum444** [comment]
**RFR**
48>>	The placement of ticket price is higher as compared to template provided by designer
37>> The Get Ticket Button should be white background and orange text (sorry for changing it again but looks very cluttered)
36>>	Next Page button not working at the bottom.
35>> The buttons showing number of pages at the bottom should be as per the template.
2>> When I sign in, the dividing orange line, my name and the cart should be moved slightly towards right.
7 >> Please make "D" capital for WelaDela.
11>> Please add verified visa with master card. They should be in one line.
17>> Can you please also redirect My Rewards and Purchase History pages View Cars button to new site too?
21>> It should open mailbox on clicking on it.
29>> Space is not equal from above and lower of icon.
30>> In whatsapp number, format should be +971 50 456 3832
As he has asked for whatsapp number format, not for landline.
33>> View Cars - Grid
In all ads displayed on main View Cars Page, the ad description should be long enough to cover full length of the image above it.

**Comment by ZarshaB** [comment]
**Reopened** 
30>> I think space is more between 971 and 50. 
![image](https://user-images.githubusercontent.com/35685146/54183778-d0466b00-44c6-11e9-8fe0-f5637d8efef5.png)

**Comment by ZarshaB** [comment]
**Reopened**
33>>View Cars - Grid
In all ads displayed on main View Cars Page, the ad description should be long enough to cover full length of the image above it.

**It is still not covering full length of image. **
![image](https://user-images.githubusercontent.com/35685146/54183907-21eef580-44c7-11e9-8d59-1a898a268dc6.png)

**Comment by ZarshaB** [comment]
34>> In Our Featured Cars, ads are appearing like this. And also new ads are not appearing here if I have marked them featured. 
![image](https://user-images.githubusercontent.com/35685146/54184098-a04b9780-44c7-11e9-8313-99c532cbf68a.png)

**Comment by ZarshaB** [comment]
35>> Please let me rephrase for the grey bar. The grey bar with Submit an Ad button should one all main pages which are "How it Works", View Cars, Detailed car view page, Results, Contact Us, FAQs, T&Cs, Point Redeem Policy and Privacy Policy. However, there shouldn't be any headings that have been given on the left of that grey bar. Other than these pages, there shouldn't be any grey bar nor any Submit an Ad button.

**Comment by ZarshaB** [comment]
36>> Let me clarify the font thing in detail as well. The font used by designer is also Lato as pointed out by that image. You have also used Lato. WhatFont tool is showing something different. So lets forget the WhatFont tool. Stick to Lato. Secondly, if we zoom out all the pages on http://template.weladela.com/ to either 67% or 75%, it looks exactly the same as designer's images. Yes, the placement of buttons is effected and so is the margin on the sides and between the each ad and similarly on other pages as well. So you have to find a way out to have the same sharpness by reducing the size without effecting the placement of above mentioned things. Im attaching zoomed out images of the site.

**Comment by Anum444** [comment]
**RFR**
13>> Login
Please remove the boxes of New Customer and Returning Customer. Instead use a green line under New Customer (and green color for this New Customer heading) and orange line under Returning customer headings (and orange color font for Returning customer heading). Reference of what I’m looking at can be taken from Car Description page where we have “Description” and “details” tabs and have smart lines under them. Attaching Image #4 for reference.
14>> Same as above when Forgot password link is clicked
30>> I think space is more between 971 and 50.

**Comment by ZarshaB** [comment]
**Reopnend**
58>> If I manually enter values then after sometime it redirects me to this error page.
![image](https://user-images.githubusercontent.com/35685146/54214565-49b27d80-4508-11e9-85ec-a046f5b946e7.png)

**Comment by Anum444** [comment]
**RFR**
34>> In Our Featured Cars, ads are appearing like this from the center. It should start from left.
58>> If I manually enter values then after sometime it redirects me to this error page.
32>> Now filter is going below 100. Can you please remove this blue outline around price filter button?
Register Button does not work when I fill the form and try to submit it.
46>> View Cars - List
In all ads displayed on main View Cars Page, ad description should be fully shown. However, if its way too long then it should be stopped till it reaches parallel to start of Get Ticket button.

**Comment by Anum444** [comment]
**RFR**
35>> Please let me rephrase for the grey bar. The grey bar with Submit an Ad button should one all main pages which are "How it Works", View Cars, Detailed car view page, Results, Contact Us, FAQs, T&Cs, Point Redeem Policy and Privacy Policy. However, there shouldn't be any headings that have been given on the left of that grey bar. Other than these pages, there shouldn't be any grey bar nor any Submit an Ad button.
12>> Login
Please refer to image # 3. Remove this heading and all similar headings (in grey bar) on all pages like View Cars, My Account etc.

**Comment by Anum444** [comment]
**RFR**
54>> Car Detail
The quantity button is working in odd numbers and goes to negative as well.
55>> Car Detail
The columns of Details should be reduced in width.

**Comment by ZarshaB** [comment]
**Reopened** 
35>> Cart page should not contain this bar. 
![image](https://user-images.githubusercontent.com/35685146/54271931-5dabbd00-45a4-11e9-97ff-efbf51f33e8f.png)

Also please remove headings from all pages.
![image](https://user-images.githubusercontent.com/35685146/54271994-803dd600-45a4-11e9-8bfc-df3ac8dbd064.png)
![image](https://user-images.githubusercontent.com/35685146/54272008-88961100-45a4-11e9-84df-a014886cc896.png)
